packages/middleware-retry/src/defaultStrategy.ts

```diff
@@ -98,6 +99,17 @@ export class StandardRetryStrategy implements RetryStrategy {
     let attempts = 0;
     let totalDelay = 0;
 
+    const maxAttempts = await (async () => {
```

Comment:
Minor: should we pull this out into its own function?

Reply:
Moved in [commit](https://github.com/aws/aws-sdk-js-v3/pull/1286/commits/d1a896b106b7167d9a396e7a6bb451f6687a2268)